src/App.css

```diff
@@ -1,9 +1,209 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
 .App-header {
   background-color: #e24e54;
   padding: 20px;
   color: white;
+  display: flex;
+  height: 100px;
+  flex-direction: row;
+  align-content: center;
+}
+
+.app-icon {
+  align-items: center;
+  flex-basis: 10%;
+  border: 1px solid black;
+}
+
+.app-search {
+  display: flex;
+  align-items: center;
+  flex-basis: 50%;
+}
+
+.app-signin {
+}
+
+.app-login {
 }
 
 .App-intro {
   font-size: large;
 }
+
+h2 {
+  font-size: 40px;
+  line-height: 47px;
+}
+
+p {
+  font-size: 14px;
+  line-height: 16px;
+  margin-bottom: 40px;
+}
+
+a {
+  color: #e24e54;
+}
+
+.container {
+  flex-direction: column;
+  background: #eee;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.contributed {
+  padding: 5px;
+  border-right: 1px solid gray;
+}
+
+.contributed1 {
+  padding: 5px;
+  border-right: 1px solid gray;
+}
+
+.contributed2 {
```

Comment:
take a look at :first-child, :nth-child, and :last-child. They might be able to help in this type of situation